event-streams/src/main/java/com/redhat/cloud/platform/playbook_dispatcher/RunEventTransform.java

```diff
@@ -143,6 +145,31 @@ Payload buildRunPayload(Struct input) {
         payload.setCreatedAt(input.getString("created_at"));
         payload.setUpdatedAt(input.getString("updated_at"));
 
+        // If the option fields do not exist, set them to empty strings
+        try {
+            payload.setName(input.getString("name"));
+        } catch (DataException e) {
+            payload.setName("");
```

Comment:
if the field is not defined then it should be represented as `"name": null` in the event stream